products/ckeditor.md

```diff
@@ -0,0 +1,35 @@
+---
+title: CKEditor
+category: framework
+permalink: /ckeditor
+releaseColumn: true
+activeSupportColumn: true
+extendedSupportColumn: true
+releasePolicyLink: https://ckeditor.com/docs/ckeditor5/latest/updating/versioning-policy.html
+iconSlug: ckeditor4
+
+releases:
+-   releaseCycle: "5"
+    eol: false
+    support: true
+    latest: "41.1.0"
+    link: https://github.com/ckeditor/ckeditor5/releases/tag/v41.1.0
+    latestReleaseDate: 2024-02-07
+    extendedSupport: false
+
+-   releaseCycle: "4"
+    eol: 2023-06-30
+    extendedSupport: 2026-12-01
+    lts: true
+    support: false
+    latest: "4.24.0"
+    link: https://ckeditor.com/cke4/release/CKEditor-4.24.0-LTS
+    latestReleaseDate: 2024-02-07
+    lts: true
+
+---
+
+> [CKEditor](https://ckeditor.com/) is an open-source WYSIWYG library written in JavaScript.
+
```

Comment:
Is it OK to assume that only the latest release is supported ? If so, can you document it ?

Reply:
I don't think that is true. CKE 5 was published in 2018 and supported alongside CKE 4 for several years.
I could not find an official release policy.